iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -45,6 +52,19 @@ public TemperatureControllerSample(DeviceClient deviceClient, ILogger logger)
 
         public async Task PerformOperationsAsync(CancellationToken cancellationToken)
         {
+            // Set handler to receive and respond to connection status changes.
+            _deviceClient.SetConnectionStatusChangesHandler(async (status, reason) =>
+            {
+                _logger.LogDebug($"Connection status change registered - status={status}, reason={reason}.");
+
+                // Call GetWritablePropertiesAndHandleChangesAsync() to get writable properties from the server once the connection status changes into Connected.
+                // This can get back "lost" property updates in a device reconnection from status like Disconnected_Retrying or Disconnected.

```

Comment:
nit: The "like" in "...property updates in a device reconnection from status _like_ Disconnected_Retrying or Disconnected." can be removed.